inception-workload-dynamic/src/main/java/de/tudarmstadt/ukp/inception/workload/dynamic/management/DynamicWorkloadManagementPage.java

```diff
@@ -250,10 +265,10 @@ private void actionShowInfoDialog(AjaxRequestTarget aTarget, IModel<SourceDocume
     public List<String> listUsersInProgressForDocument(SourceDocument aDocument)
     {
         List<String> result = new ArrayList<>();
-        for (AnnotationDocument anno: documentService.
-            listAnnotationDocuments(currentProject.getObject())) {
+        for (AnnotationDocument anno : documentService
```

Comment:
Slow - filtering should happen at SQL level in a backend service bean, e.g. documentService.